### DISCUSSION TITLE
Favorites select multiple

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#183] [Q&A] [answered]
Could be implemented to select many locations at once by hitting on the symbol one after the other and the assign a group for them all?

**Comment by devemux86** [comment]
@DolomitesTom thanks for the idea!

Do you mean in the 2nd tab of favorites, select multiple favorites at once and move them to a new group?

**Reply by DolomitesTom** [reply]
Yes

**Reply by devemux86** [reply]
I will see about adding check boxes to favorites.
Then we can select multiple favorites and perform actions on all of them.

**Comment by devemux86** [comment]
Multiple selection in the favorites screen will be available in the next Cruiser.

Tap to select / deselect favorites, the icon changes and the number of selected is shown in the title.
Then with long press we can perform actions on the selected favorites, like select a different group.

<img src="https://github.com/devemux86/cruiser/assets/3484020/ae687d0c-87b2-4bc0-aed6-0b109e140376" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/8bdc053a-fd67-4388-ad3e-5bcf5bcecb31" width="50%"/>

**Comment by devemux86** [comment, marked as answer]
Multiple selection in favorites screen is available in [Cruiser 3.0.22](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

Tap to select / deselect favorites, please see the above images: https://github.com/devemux86/cruiser/discussions/183#discussioncomment-6047959

**Reply by DolomitesTom** [reply]
works perfect thx
